src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/blockeditor/blockeditor.html

```diff
@@ -0,0 +1,56 @@
+<div class="umb-block-editor"  ng-controller="Umbraco.Editors.BlockEditorController as vm">
+
+    <ng-form name="blockForm" val-form-manager>
+        <umb-editor-view umb-tabs ng-if="!page.loading">
+
+            <umb-editor-header
+                name="vm.model.title"
+                name-locked="true"
+                navigation="vm.tabs"
+                hide-alias="true"
+                hide-icon="true"
+                hide-description="true">
+            </umb-editor-header>
+
+            <div class="umb-editor-container umb-panel-body umb-scrollable row-fluid">
+
+                <!-- Missing Deleted Message Bar (Displayed when viewing node in recycle bin), but Blocks dosnt go into trash-bin jet...
+                    <div ng-show="vm.content.trashed" class="umb-editor--trashed-message">
+                        <i class="icon icon-trash"></i> <localize key="content_nodeIsInTrash">This item is in the Recycle Bin</localize>
+                    </div>
+                -->
+
+                <div class="umb-pane">
+                    <umb-editor-sub-views sub-views="vm.tabs" model="vm.model">
+                    </umb-editor-sub-views>
+                </div>
+
+            </div>
+
+            <umb-editor-footer>
+
+                <!-- Missing breadcrumbs -->
```

Comment:
They are not there, but they might not need to be here either. Properly first necessary when we start having Blocks in the Database and thereby having shared blocks. Or if we one day enable linking between Blocks. Currently, the only thing indicating where this block is located in the layers you have navigated to get there. This is also the only way to get there, but then the other things become true the breadcrumbs can help clarify where you are at.

Do you want it removed? I do find it a good comment for future development. Its because the code originates from the content editor, this is basically a copy of it, working with Blocks instead. So i guess its good to point out that it would belong here to match contentEditor.

Reply:
That's fine, lets leave it then :)